src/ray/gcs/redis_async_context.cc

```diff
@@ -56,6 +56,9 @@ Status RedisAsyncContext::RedisAsyncCommand(redisCallbackFn *fn, void *privdata,
   {
     // `redisvAsyncCommand` will mutate `redis_async_context_`, use a lock to protect it.
     std::lock_guard<std::mutex> lock(mutex_);
+    if(!redis_async_context_) {
+      return Status::NotImplemented("...");
```

Comment:
What's this for?

Reply:
Ah, this was for debugging purposes. Removed it.